Veeam/PRTG-VeeamBRStats.ps1

```diff
@@ -205,7 +238,7 @@ Write-Host "<result>"
                "</result>" 
 $Count = $failsSessionsBk.Count
 Write-Host "<result>"
-               "<channel>Failes-Backups</channel>"
```

Comment:
Renaming the XML Sections is a problem for existing User.
Please remove all these Changes.